service/gcs/run_unittests

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex 
```

Comment:
I'm kind of confused. Is this file intended to be a replacement for setup_test_env? We probably want setup and test running to be two separate stages. Running the setup script would go under the `install:` field in the Travis YML file. Then test running would basically just be running ginkgo, and wouldn't even need its own script at this point.